### PR TITLE
Fixed the Errors

### PR DESCRIPTION
**Pull request by khmdagal**
I fixed errors on the answers of the in Linus's homework in (homework.md) file.

**Liverpool NW4 - Khadar M Dagal  -  Linus's homework in (homework.md) file. - week 1**

# Your Details

- Your Name: Khadar M Dagal
- Your City: Liverpool
- Your Slack Name: Khadar M Dagal

# Homework Details

- Module: Help me with my kid's homework! (1 hour)

### CODE REVIEW

**Comment by musayuksel** [comment]
Hi Khadar,
Perfectly corrected answers and they all look right to me...

**Comment by ahmd202** [comment]
Nice Work Khadar. Good job.

File: homework.md [hunk 1]
```diff
@@ -3,111 +3,109 @@
...
+ h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
-
-A programming language used on the internet
+Answer 7: JavaScript is a programming language that is used for web developments.
```
Comment (zeinaji) [inline homework.md:42]: Also worth mentioning that it's used to add special effects and dynamic behaviour to the webpage :)

File: homework.md [hunk 2]
```diff
@@ -3,111 +3,109 @@
...
+Answer 17: Commit Messages makes the communication and collaboration more effecive and efficient, becasue Commit Messeages makes clear, why changes happened in the first place 
+           so it will be easier other developers to understand the changes. 
+           
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+Answer 17 : you can find more helpfull tips just by searching google simply by typing the quesion, also you can find more offical help and tips from https://git-scm.com/docs/git-help.
+
```
Comment (zeinaji) [inline homework.md:111]: You could also type `git --help` on the terminal

**Review by zeinaji — COMMENTED** [review]
Great work Khadar 🙂

**Pull request closed without merging**